js/editPairForm.js

```diff
@@ -242,7 +242,7 @@ $(document).ready(function () {
 
     // Custom validation methods
     $.validator.addMethod("supernumeric", function (value, element) {
-      return this.optional(element) || /^[A-Za-z0-9_,-.' ]+$/.test(value);
+      return this.optional(element) || /^[A-Za-z0-9_,-.' /s ]+$/.test(value);
```

Comment:
no need to add a `/s` here since ` ` (space is included after `'`) is present in the regex.

Reply:
Remain all looks fine!! Just make this change and I will merge it :)